manifests/kustomization.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+commonAnnotations:
+  goharbor.io/cluster-version: "0.5.0"
+  goharbor.io/template-version: "20200815"
+  goharbor.io/template-engine: "Kustomization"
+
+resources:
+  - namespaces/namespaces.yaml
+  - https://raw.githubusercontent.com/spotahome/redis-operator/master/example/operator/all-redis-operator-resources.yaml
+  - core-operator/all-core-operator-resources.yaml
+
+bases:
+  - ../config/default
+  - github.com/zalando/postgres-operator/manifests
+  - github.com/minio/operator?ref=8d6919ae93fef84d12d5663128dd2ea5d9eaff13
```

Comment:
please use the latest version of minio-operator

Reply:
udated